sdk/storage/Azure.Storage.DataMovement.Blobs/src/BlobJobPlanExtensions.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Text;
+using Azure.Storage.Blobs.Models;
+
+namespace Azure.Storage.DataMovement.Blobs
+{
+    internal static class BlobJobPlanExtensions
```

Comment:
Probably change this to partial class `JobPlanExtensions` instead to be merged with shared one.